src/planner/Admin.cpp

```diff
@@ -72,5 +72,94 @@ std::unique_ptr<cpp2::PlanNodeDescription> GetConfig::explain() const {
     addDescription("name", name_, desc.get());
     return desc;
 }
+
+std::unique_ptr<cpp2::PlanNodeDescription> CreateNode::explain() const {
+    auto desc = SingleDependencyNode::explain();
+    addDescription("ifNotExist", util::toJson(ifNotExist_), desc.get());
+    return desc;
+}
+
+std::unique_ptr<cpp2::PlanNodeDescription> DropNode::explain() const {
+    auto desc = SingleDependencyNode::explain();
+    addDescription("ifExist", util::toJson(ifExist_), desc.get());
+    return desc;
+}
+
+std::unique_ptr<cpp2::PlanNodeDescription> CreateUser::explain() const {
+    auto desc = CreateNode::explain();
+    addDescription("username", *username_, desc.get());
+    addDescription("password", *password_, desc.get());
+    return desc;
+}
+
+std::unique_ptr<cpp2::PlanNodeDescription> DropUser::explain() const {
+    auto desc = DropNode::explain();
+    addDescription("username", *username_, desc.get());
+    return desc;
+}
+
+std::unique_ptr<cpp2::PlanNodeDescription> UpdateUser::explain() const {
+    auto desc = SingleDependencyNode::explain();
+    addDescription("username", *username_, desc.get());
+    addDescription("password", *password_, desc.get());
```

Comment:
Shall we return password without any mask to console ?

Reply:
Maybe we could add note in document.

Reply:
> Shall we return password without any mask to console ?

Agree, can't return the password directly. need to modify it.

Reply:
This won't execute in fact, so the password will not stored.

Reply:
Had fixed.